person.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'student'
+
+class Person
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return if @age >= 18 && parent_permission
+
+    true
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ optional ] Hi it's always better to follow best practices(DRY). in the `can_user_services` function, you could replace the  `if @age >= 18` with the private function you created `of_age?`

Reply:
@m05e5  Thanks for the suggestion but can I use the method `of_age?` inside `can_user_services` before initializing or calling it